suite/tests/client-interface/inline.c

```diff
@@ -30,14 +30,16 @@
  * DAMAGE.
  */
 
+#include "configure.h"
+
 /* Export instrumented functions so we can easily find them in client.  */
 #ifdef WINDOWS
 # define EXPORT __declspec(dllexport)
 #else /* UNIX */
 # define EXPORT __attribute__((visibility("default")))
 #endif
 
-/* List of instrumented functions. */
+#ifdef X86
 #define FUNCTIONS() \
```

Comment:
style: space after #

Reply:
Odd that the style checks did not catch that. I've fixed those.